lit/SwiftREPL/FoundationTypes.test

```diff
@@ -0,0 +1,29 @@
+// Test that we can correctly print (resilient) Foundation
+// types when they're stored in REPL-defined globals.
+
+// REQUIRES: darwin
+
+// RUN: %lldb --repl < %s | FileCheck %s
+
+import Foundation
+
+let u = URL(string: "http://www.apple.com")!
+// CHECK:      u: URL = "http://www.apple.com"
+
+
+let n = Notification(name: Notification.Name("abc"), object: Double(42.0), userInfo: ["hello":"world"])
+
+// CHECK:      n: Notification = {
+// CHECK-NEXT:   name = "abc"
+// CHECK-NEXT:   object = 42
+// CHECK-NEXT:   userInfo = 1 key/value pair {
+// CHECK-NEXT:     [0] = {
+// CHECK-NEXT:       key = {
+// CHECK-NEXT:         _box = {
+// CHECK-NEXT:           _baseHashable = "hello"
+// CHECK-NEXT:         }
+// CHECK-NEXT:       }
+// CHECK-NEXT:       value = "world"
+// CHECK-NEXT:     }
+// CHECK-NEXT:   }
+// CHECK-NEXT: }
```

Comment:
I made this made this mistake myself  just last week:

You need to have something in the CHECK lines that does not appear verbatim in the file, or else you will match the comment in the REPL *input* rather than the output produced later on!

For example, just doing `CHECK:      n: Notification {{=}} {` in that one line is sufficient to guard against this.

Reply:
Otherwise, thanks, this great! :-)

Reply:
Thanks, fixed!